package.json

```diff
@@ -16,7 +16,7 @@
     "ejs": "^3.1.9",
     "ejs-mate": "^4.0.0",
     "express": "^4.18.2",
-    "mongoose": "^7.3.4"
+    "mongoose": "^7.4.1"
   },
```

Comment:
Why are updating mongoose version here? Any specific reason if no please revert it to 7.3.4, you can use the command `npm i mongoose@7.3.4`